client/src/main/scala/com.gu.contentapi.client/utils/QueryStringParams.scala

```diff
@@ -5,7 +5,11 @@ import java.net.URLEncoder
 object QueryStringParams {
   def apply(parameters: Iterable[(String, String)]) = {
     def encodeParameter(p: String): String = p match {
```

Comment:
This `match` is now superfluous.  The need for it was removed in 50ee718fc1c6ee06e9853a400ecb4045710daa96.

Reply:
good catch!